jib-cli/src/test/resources/buildfiles/projects/allProperties/jib.yaml

```diff
@@ -0,0 +1,58 @@
+# this buildfile doesn't necessarily work, just useful for testing parsing and translation
+apiVersion: jib/v1alpha1
+kind: BuildFile
+
+# "FROM" with detail for manifest lists or multiple architectures
+from:
+  image: "ubuntu"
+  # optional: if missing, then defaults to `linux/amd64`
+  platforms:
+    - architecture: "arm"
+      os: "linux"
+      os.version: "a"
```

Comment:
I realized, these values "os.version", "os.features", "variant", "features" are all ignored in the current implementation, because we don't support it at lower levels of jib.

Reply:
this is fixed